packages/location-management/src/components/LocationUnitGroupAddEdit/tests/Form.test.tsx

```diff
@@ -0,0 +1,227 @@
+import { mount } from 'enzyme';
+import React from 'react';
+import { history } from '@onaio/connected-reducer-registry';
+import { Provider } from 'react-redux';
+import { MemoryRouter, Route, Router } from 'react-router';
+import { notification } from 'antd';
+import flushPromises from 'flush-promises';
+import fetch from 'jest-fetch-mock';
+import { store } from '@opensrp/store';
+
+import Form, { onSubmit } from '../Form';
+import * as fixtures from './fixtures';
+import { act } from 'react-dom/test-utils';
+
+describe('Location-module/Form', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('renders without crashing', () => {
+    const wrapper = mount(
+      <Provider store={store}>
+        <Router history={history}>
+          <Form />
+        </Router>
+      </Provider>
+    );
+    expect(wrapper.props()).toMatchSnapshot();
```

Comment:
For this test case we can also do away with the snapshot

Reply:
done removed that snapshot it was just to check if we have form or not